pkgs/development/ruby-modules/gem-config/default.nix

```
@@ -379,6 +379,10 @@ in
     '';
   };
 
+  mimemagic = attrs: {
+    FREEDESKTOP_MIME_TYPES_PATH="${shared-mime-info}";
```

```suggestion
    FREEDESKTOP_MIME_TYPES_PATH="${shared-mime-info}/share/mime/packages/freedesktop.org.xml";
```

According to https://github.com/mimemagicrb/mimemagic#dependencies it should point to the xml file but I did not test this.